### PR TITLE
TAJO-2073: Upgrade parquet-mr to 1.8.1.

### PR DESCRIPTION
**Pull request by jinossy**

### CODE REVIEW

File: tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/parquet/ParquetScanner.java [hunk 1]
```diff
@@ -78,6 +96,7 @@ public Tuple next() throws IOException {
    */
   @Override
   public void reset() throws IOException {
+    throw new TajoRuntimeException(new UnsupportedException());
```
Comment (jihoonson) [inline tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/parquet/ParquetScanner.java]: UnimplementedException looks more proper.

File: tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/thirdparty/parquet/InternalParquetRecordReader.java [hunk 2]
```diff
@@ -70,37 +81,50 @@
...
       if (current != 0) {
-        long timeAssembling = System.currentTimeMillis() - startedAssemblingCurrentBlockAt;
-        totalTimeSpentProcessingRecords += timeAssembling;
-        if (DEBUG) LOG.debug("Assembled and processed " + totalCountLoadedSoFar + " records from " + columnCount + " columns in " + totalTimeSpentProcessingRecords + " ms: " + ((float) totalCountLoadedSoFar / totalTimeSpentProcessingRecords) + " rec/ms, " + ((float) totalCountLoadedSoFar * columnCount / totalTimeSpentProcessingRecords) + " cell/ms");
-        long totalTime = totalTimeSpentProcessingRecords + totalTimeSpentReadingBytes;
-        long percentReading = 100 * totalTimeSpentReadingBytes / totalTime;
-        long percentProcessing = 100 * totalTimeSpentProcessingRecords / totalTime;
-        if (DEBUG) LOG.debug("time spent so far " + percentReading + "% reading ("+totalTimeSpentReadingBytes+" ms) and " + percentProcessing + "% processing ("+totalTimeSpentProcessingRecords+" ms)");
+        totalTimeSpentProcessingRecords += (System.currentTimeMillis() - startedAssemblingCurrentBlockAt);
+        if (Log.INFO) {
```
Comment (jihoonson) [inline tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/thirdparty/parquet/InternalParquetRecordReader.java]: Even though these logs seem to be printed whenever a row group is fully read, I'm concerned with there will be too many logs.

File: tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/thirdparty/parquet/InternalParquetRecordReader.java [hunk 3]
```diff
@@ -138,24 +162,29 @@ public float getProgress() {
     return (float) current / total;
   }
 
-  public void initialize(MessageType requestedSchema, MessageType fileSchema,
-                         Map<String, String> extraMetadata, Map<String, String> readSupportMetadata,
+  public void initialize(MessageType fileSchema,
```
Comment (jihoonson) [inline tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/thirdparty/parquet/InternalParquetRecordReader.java]: FileMetaData also includes the file schema.

**Comment by jihoonson** [comment]
+1. I left trivial comments. Please consider them before you commit.

**Comment by jinossy** [comment]
Thanks for your review!
I'll commit it that reflects your comments

**Pull request closed without merging**